package.json

```diff
@@ -121,6 +121,7 @@
     "screener-runner": "^0.10.30",
     "simulant": "^0.2.2",
     "snyk": "^1.110.2",
+    "syncpack": "^4.3.1",
```

Comment:
this is an additional dependency introduced - lets provide a note in the dedicated PR section about that (currently there is one, but lets just make it more visible)

Reply:
> * adds [syncpack](https://github.com/JamieMason/syncpack) to align `package.json` files and our dependencies across packages

It was mentioned in the initial comment. Can you please clarify what changes do you expect?